### PR TITLE
py-find-libpython: 0.3.1

### PR DESCRIPTION
**Pull request by heerener**
Now that there's a source tarball for 0.3.1 we can get the latest version as well.

### CODE REVIEW

**Comment by heerener** [comment]
@adamjstewart I'm not emotionally attached to 0.3.0 - would it be better to remove it and keep the recipe simpler? Or is that frowned upon now that it's already out there in the wild?

**Comment by adamjstewart** [comment]
We usually keep them unless it makes the recipe more complicated, but see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions for the official guidance.

**Comment by heerener** [comment]
All right, thanks!
I have no strong desire to remove 0.3.0 either, so let's not go through the whole deprecation cycle :-)

File: var/spack/repos/builtin/packages/py-find-libpython/package.py [hunk 1]
```diff
@@ -15,6 +15,8 @@ class PyFindLibpython(PythonPackage):
     license("MIT")
 
     version("0.3.0", sha256="6e7fe5d9af7fad6dc066cb5515a0e9c90a71f1feb2bb2f8e4cdbb4f83276e9e5")
+    version("0.3.1", sha256="4dd75e54c0828cfa8e97287565d2499a6bd6216140afdf251c87a456e8e52cd3")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-find-libpython/package.py:17]: Can you sort newest to oldest?

Comment (heerener) [reply var/spack/repos/builtin/packages/py-find-libpython/package.py:17]: Sure!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**